src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -512,14 +507,14 @@ public static Exception AttachRestrictedErrorInfo(Exception e)
 
                     if (restrictedErrorInfoPtr != IntPtr.Zero)
                     {
-                        ObjectReference<IUnknownVftbl> restrictedErrorInfoRef = ObjectReference<IUnknownVftbl>.Attach(ref restrictedErrorInfoPtr, IID.IID_IRestrictedErrorInfo);
+                        using ObjectReference<IUnknownVftbl> restrictedErrorInfoRef = ObjectReference<IUnknownVftbl>.Attach(ref restrictedErrorInfoPtr, IID.IID_IRestrictedErrorInfo);

```

Comment:
Can we skip allocating this object reference entirely and just pass `restrictedErrorInfoPtr` to `GetErrorDetails` below? I mean it seems this object reference is only ever used in this scope anyway, so perhaps it's unnecessary?

Reply:
Addressed along with a couple other allocations.